packages/flutter_tools/test/src/context.dart

```diff
@@ -32,49 +34,34 @@ void testUsingContext(String description, dynamic testMethod(), {
   test(description, () async {
     AppContext testContext = new AppContext();
 
+    // Apply all overrides to the test context.
     overrides.forEach((Type type, dynamic value) {
       testContext[type] = value;
```

Comment:
I'm removing the `[]=` operator but there are still places that we want to override something in a context so I'm leaving `setVariable`